geode-memcached/src/integrationTest/java/org/apache/geode/internal/memcached/MemcachedSanctionedSerializablesServiceIntegrationTest.java

```diff
@@ -14,30 +14,36 @@
  */
 package org.apache.geode.internal.memcached;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import java.io.IOException;
 import java.net.URL;
 import java.util.Collection;
 
-import org.apache.geode.distributed.internal.DistributedSystemService;
-import org.apache.geode.distributed.internal.InternalDistributedSystem;
-import org.apache.geode.internal.InternalDataSerializer;
-import org.apache.geode.internal.classloader.ClassPathLoader;
+import org.junit.Test;
 
-public class MemcachedDistributedSystemService implements DistributedSystemService {
-  @Override
-  public void init(InternalDistributedSystem internalDistributedSystem) {
+import org.apache.geode.internal.serialization.SanctionedSerializablesService;
 
-  }
+public class MemcachedSanctionedSerializablesServiceIntegrationTest {
```

Comment:
Note: github diffs are confused... the code is correct.

MemcachedDistributedSystemService was renamed to MemcachedSanctionedSerializablesService and still exists in geode-memcached.

MemcachedSanctionedSerializablesServiceIntegrationTest is a new test that was created to help debug it when I hit a problem adding the service.